### ISSUE TITLE
Ankidroid media sync error

### ISSUE THREAD

**Issue by ghost** [kuklinistvan/docker-anki-sync-server#18]
Hey!

I am getting the same error message, but I already double-checked that there are no trailing slashes.

There is a logged request and it looks a lot like the requests made by my desktop, but it ends in 200 16 and not 200 52, if that is useful at all.

_Originally posted by @kunberg in https://github.com/kuklinistvan/docker-anki-sync-server/issues/5#issuecomment-627699269_

**Comment by kuklinistvan** [comment]
Hello there,

Can you give me the AnkiDesktop version number involved in your scenario?

Also, can you describe the exact steps you've followed to set up the server and the client? The more details you give, the better ;)

Also, can you collect these data:
* Screenshots of the desktop (particularly the error message)
* Console output of the desktop
* Server logs (`docker logs`)

Thank you!

**Comment by ghost** [comment]
Oh sorry, just realised, that I did not end up using docker, so this is probably the wrong repository. Should I head over to  tsudoko/anki-sync-server?

I followed their instructions. I also set up a reverse proxy for ssl using apache, but I had seen the error before I set that up as well.

The error message only occurs when trying to synchronize AnkiDroid. AnkiDesktop (2.2.21) is working without any problems.

I have tried both AnkiDroid 2.9.1 and 2.9.7, but the error persists.

These are the server logs when using AnkiDesktop (successfully):

```
[2020-05-13 15:54:22,249]:INFO:ankisyncd.CollectionThread[username]:Running meta(*[], **{'v': 9, 'cv': 'ankidesktop,2.1.20 (47a1bf8b),win:10'})

[2020-05-13 15:54:22,255]:INFO:ankisyncd.http:127.0.0.1 "POST /sync/meta HTTP/1.1" 200 112

[2020-05-13 15:54:22,334]:INFO:ankisyncd.CollectionThread[username]:Running begin(*[], **{'skey': 'skey'})

[2020-05-13 15:54:22,335]:INFO:ankisyncd.http:127.0.0.1 "POST /msync/begin HTTP/1.1" 200 52
```

And this is what happens when I try to use my phone:

![WhatsApp Image 2020-05-13 at 16 24 13](https://user-images.githubusercontent.com/65265124/81825146-8f1c7180-9536-11ea-80f6-066c00d8c8a2.jpeg)

```
[2020-05-13 16:18:02,238]:INFO:ankisyncd.CollectionThread[felix]:Running meta(*[], **{'v': 9, 'cv': 'ankidroid,2.9.7,android:9:ONEPLUS A3003'})

[2020-05-13 16:18:02,240]:INFO:ankisyncd.http:127.0.0.1 "POST /sync/meta HTTP/1.1" 200 112

[2020-05-13 16:18:02,435]:INFO:ankisyncd.http:127.0.0.1 "POST /mysync/begin HTTP/1.1" 200 16
```

Thank you so much!

**Comment by kuklinistvan** [comment]
> Oh sorry, just realised, that I did not end up using docker, so this is probably the wrong repository. Should I head over to tsudoko/anki-sync-server?

Well, this repository aims to support the painless installation of the private server infrastructure of one supported set of versions, which is at this time AnkiDesktop version 2.1.19 and AnkiDroid version 2.9.1.

As of writing, it is achieved through @tsudoko's `ankisyncd` v. [2.1.0](https://github.com/tsudoko/anki-sync-server/tree/2.1.0) patched with commit [2bfccf7f](https://github.com/tsudoko/anki-sync-server/commit/2bfccf7fa4af87545f009729ff6e84934c6fde0b). I do my best to [personally verify](https://github.com/kuklinistvan/docker-anki-sync-server/blob/master/Testing.md) such a container image before releasing, but errors could happen anyway - in that case, you can report that to me :)

* If you wish to have Anki server working quickly, I suggest you downgrading your clients (see the download links in the [README](https://github.com/kuklinistvan/docker-anki-sync-server#tested-and-works-on)) and using this Docker image.
* On the other hand, if you wish to experiment with the latest versions or manually installing the server for some other reason, then yes, your next stop is probably the ["upstream" repository](https://github.com/tsudoko/anki-sync-server). In the latter case, please post all these information as it contains useful clues for reproduction!

Good luck!

**Comment by ghost** [comment]
Great, thank you!
